src/Symfony/Component/Routing/Attribute/Route.php

```diff
@@ -201,6 +207,28 @@ public function getEnv(): ?string
     {
         return $this->env;
     }
+
+    public function getAliases(): array
+    {
+        return $this->aliases;
+    }
+
+    /**
+     * @param list<string>|string $aliases
```

Comment:
I suggest using `string[]` instead of `list<string>` to be less restrictive on the input. This will be consistent with other methods (and with the constructor where you don't ask for a list) and the code consuming `getAliases` does not even read the keys so it is fine with any array of strings, not just with lists.